l3kernel/l3tl.dtx

```diff
@@ -1269,6 +1269,13 @@
 %
 % \subsection{Functions}
 %
+% \begin{macro}{\__kernel_tl_set:Nx,\__kernel_tl_gset:Nx}
+%    \begin{macrocode}
+\cs_new_eq:NN \__kernel_tl_set:Nx \cs_set_nopar:Npx
+\cs_new_eq:NN \__kernel_tl_gset:Nx \cs_gset_nopar:Npx
```

Comment:
Could you please document these (as trivial as they are) in `l3kernel-functions.dtx`?

Reply:
…and replace all(?) uses of `\tl_set:Nx` by `\__kernel_tl_set:Nx` (and same for `\tl_gset:Nx`).  This would speed up slightly `\tl_concat:NNN`, `\tl_replace_...`, `\tl_trim_spaces:N`, `\tl_reverse:N`.  In fact, maybe we should make that change much more broadly in `l3kernel`, which would for instance speed up `\clist_set:Nn` or `\seq_put_left:Nn`.

Reply:
@blefloch I'd say yes, but this might break some assumptions of `l3debug` of macros using `\tl_set:Nx`/`\tl_gset:Nx` internally for stuff like the scope checks.

Reply:
`l3debug` completely destroys performance by adding the checks, so since the “user interface” of `\__kernel_tl_set:Nx` and `\tl_set:Nx` are the same, I think it should be fine to restore the old definition of `\tl_set:Nx` and define `\__kernel_tl_set:Nx` in terms of that.  Or even easier, just add `\__kernel_tl_set:Nx` to be patched by `l3debug` the same way `\tl_set:Nx` is.